src/Uno.Extensions.Reactive/Collections/Facades/Map/ConverterEqualityComparer.cs

```diff
@@ -22,10 +22,16 @@ public ConverterEqualityComparer(IConverter<TFrom, TTo> converter, IEqualityComp
 	}
 
 	/// <inheritdoc />
-	public bool Equals(TTo left, TTo right) 
-		=> _comparer.Equals(_converter.ConvertBack(left), _converter.ConvertBack(right));
+	public bool Equals(TTo? left, TTo? right)
+		=> (left, right) switch
+		{
+			(null, null) => true,
+			(null, _) => false,
+			(_, null) => false,
+			_ => _comparer.Equals(_converter.ConvertBack(left), _converter.ConvertBack(right))
+		};
```

Comment:
Potential perf hit.

Reply:
Those are ref equals comparison, it should not have any perf hit.

> When you match an expression against null, the compiler guarantees that no user-overloaded == or != operator is invoked.